openid-4-verifiable-presentations-1_0.md

```diff
@@ -1201,6 +1201,8 @@ The Response Mode is defined in accordance with [@!OAuth.Responses] as follows:
 
 The following new Authorization Request parameter is defined to be used in conjunction with Response Mode `direct_post`: 
 
+If the Response Endpoint receives a request in the direct_post response mode that is structurally valid and complies with the submission requirements of the presentation definition, it MUST respond with an HTTP status code 200, indicating that the request was successfully processed in terms of protocol compliance. If the submitted credential is invalid (e.g., expired, revoked, or does not meet the verifier’s functional requirements), the Response Endpoint SHOULD respond with an HTTP status code 400 (Bad Request). The 400 response MUST include an `error` parameter with a value such as `invalid_credential` or `functional_requirements_not_met`, and MAY include a `redirect_uri` to allow the wallet to resubmit with different credentials. The `error_description` parameter MAY provide additional details to assist the wallet in correcting the submission.
```

Comment:
there is no more submission requirements, since there is only DCQL. 

also this text should be in section 8.5 https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-8.5